Forking procedure (#1)
grafana/business-charts: README.md

# Business Charts for Grafana
[![CI](https://github.com/grafana/business-charts/actions/workflows/push.yml/badge.svg)](https://github.com/grafana/business-charts/actions/workflows/push.yml)
[![CD](https://github.com/grafana/business-charts/actions/workflows/publish.yml/badge.svg)](https://github.com/grafana/business-charts/actions/workflows/publish.yml)
[![License](https://img.shields.io/github/license/grafana/business-charts)](LICENSE)

>This project was originally contributed by [VolkovLabs Business Charts](https://github.com/volkovlabs/business-charts) - thanks for all your great work!
>
>We have republished under the same plugin ID, keeping the community signature. This means you can simply update your plugin version. A new ID would have required manual updates to your dashboards. For additional information on the changes, see the [Notices](/NOTICES).

> This project is now maintained by Grafana Labs. Pull requests are welcome and will be reviewed on a best-effort basis.

Reviewer: @natellium, @imatwawana - is this something that we want to address now or should we think more about it?

# Business Charts for Grafana
>This project was originally contributed by [VolkovLabs Business Charts](https://github.com/volkovlabs/business-charts) - thanks for all your great work!
>

> >We have republished under the same plugin ID, but with the Grafana signature. This means you can simply update your plugin version. A new ID would have required manual updates to your dashboards. We changed the signature to Grafana to reflect the change in publisher and so that it is clear we are not impersonating the original authors. For additional information on the changes, see the [Notices](/NOTICES).

Reviewer 2: ```suggestion
>We have republished under the same plugin ID, keeping the community signature. This means you can simply update your plugin version. A new ID would have required manual updates to your dashboards. For additional information on the changes, see the [Notices](/NOTICES).
```

> ⚠️ This project is not actively maintained by Grafana Labs. Pull requests are welcome and will be reviewed on a best-effort basis. Contact integrations@grafana.com if you are interested in taking on this project longer term. We will be happy to work with and eventually hand over to people who are interested in maintaining it again.

Reviewer 2: ```suggestion
This project is now maintained by Grafana Labs. Pull requests are welcome and will be reviewed on a best-effort basis.
```
